SETUP.md

```diff
@@ -82,8 +93,12 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
           GIT_COMPLETION_CHECKOUT_NO_GUESS=1
           autoload -Uz compinit && compinit
           ```
+        * fix any problems with compinit:
+          ```
+          compaudit | xargs chmod g-w
+          ```
     </details>
-1. Install Homebrew: `ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+1. Install Homebrew: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"`
```

Comment:
when I ran the old command, it told me it was deprecated and that I should run this command instead.